### DISCUSSION TITLE
Help! How to use swiper in Nextjs Project using NPM package.

### DISCUSSION THREAD

**Discussion by koushikpuppala** [nolimits4web/swiper#5904] [Q&A]
Hello, I'm building the Nextjs project with typescript. I'd want to use Swiper in that, however it doesn't function when I try to install it. Is there an example that works well that I can use as a reference?

**Comment by Arhell** [comment]
https://swiperjs.com/react#installation
https://stackoverflow.com/questions/71751179/swiper-js-dosnt-work-with-next-js-v-12-1-4
